components/camel-solr/src/main/docs/solr-component.adoc

```diff
@@ -0,0 +1,170 @@
+= Solr Component
+:doctitle: Solr
+:shortname: solr
+:artifactid: camel-solr
+:description: Perform operations against Apache Lucene Solr.
+:since: 2.9
```

Comment:
This is a new component so the first version should be 4.8.0 and not 2.9

Reply:
fixed